index.html

```diff
@@ -159,15 +159,16 @@ <h4 class="">Skills</h4>
         </section>
         
         
-    </footer>
-        <h2 class="head-contact">
-        <form action="" class="form-contact">
-            <input type="text" placeholder="  Full name" name="name" id="name">
-            <input type="email" placeholder="  Email address" name="email" id="email">
-            <textarea name="message" id="message" cols="30" rows="10">  Enter text here</textarea>
+    <footer class="ContacMe">
+        <h2 class="head-contact">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
```

Comment:
- Hey, this content should be a paragraph and not header content. please, let's use the right HTML element to improve our markup